src/libraries/System.Drawing.Common/src/System/Drawing/BufferedGraphicsContext.Windows.cs

```diff
@@ -304,7 +304,7 @@ private IntPtr CreateCompatibleDIB(IntPtr hdc, IntPtr hpal, int ulWidth, int ulH
                 // Create the DIB section. Let Win32 allocate the memory and return
                 // a pointer to the bitmap surface.
                 hbmRet = SafeNativeMethods.CreateDIBSection(new HandleRef(null, hdc), ref pbmi, NativeMethods.DIB_RGB_COLORS, ref ppvBits, IntPtr.Zero, 0);
-                Win32Exception ex = null;
+                Win32Exception? ex = null;
                 if (hbmRet == IntPtr.Zero)
                 {
                     ex = new Win32Exception(Marshal.GetLastWin32Error());
```

Comment:
Just curious why it is not just throw here directly

Reply:
Yeah it's strange. Wouldn't want to change it in this PR though.